src/js/AppContainer.jsx

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+var api = require('./api');
+var UserSelect = require('./UserSelect.jsx');
+var UserFiHeading = require('./UserFiHeading.jsx');
+
+var AppContainer = React.createClass({
+
+  getInitialState: function(){
+    return {
+      user: {
+        name: '',
+        institutions: []
+      }
+    }
+  },
+
+  selectCallback: function(e){
+    var user = e.target.value;
+    var self = this;
+    console.log('selecting user: ', user);
+    api.getInstitutions(user, function(institutions){
+      self.setUser({name: user, institutions: institutions});
+    });
+  },
+
+  setUser: function(user){
+    this.setState({user: user});
+  },
+
+  render: function() {
+    return (
+      <div>
+        <UserSelect callback={this.selectCallback}/>
+        <UserFiHeading institution={{}} year="2017" user={this.state.user.name}/>
```

Comment:
Acttttually, the FiStatusHeader is rendered in `AppContainer` unconditionally, which is rendered on every route, so the route would need to passed in at a higher level.

I like the idea of the route in the URL, and we could do that, but it would definitely need to be encoded with encodeURIComponent.. Thoughts?